recipes/emu-pca/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "emu-pca" %}
+{% set version = "v1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: {{ version|replace("v", "") }}
+
+source:
+  url: 'https://github.com/Rosemeis/emu/archive/refs/tags/{{ version }}.tar.gz'
+  md5: b79e9a2f72b8b83361a5542edc1c4ca7
+
+build:
+  number: 0
+  skip: true # [py<30]
```

```suggestion
  skip: true # [py<30]
```
There is no need of this.

---

```suggestion
```
Happy to remove it then. :)